### PR TITLE
Akash - solution

### PR DESCRIPTION
**Pull request by akash350poria**

### CODE REVIEW

File: NEWTASK1.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+				},
+				{
+					name:'Up',
+					license: [2007,2011]
+				},
+				{
+					name:'Cars',
+					license: [2006,2009]
+				}
+			];
```
Comment (ankushdharkar) [inline NEWTASK1.js:23]: Move to a separate file please

File: NEWTASK1.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+function yearWithMaxNumOfMovies(movieList){
```
Comment (ankushdharkar) [inline NEWTASK1.js:25]: Break down this giant function into smaller logical pieces, kind of like: https://github.com/Real-Software-Dev-Squad/Max-Movies-In-Years/pull/2#issuecomment-599252338

File: NEWTASK1.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+//Create a list with all the years of different movie license
+function combinedYearsArray(movieList){
+	var allYears=[];
+	for(var i=0;i<movieList.length;i++){
+		for(var j=0;j<movieList[i].year_range.length;j++){
+			allYears.push(movieList[i].year_range[j]);
+		}
+	}
+	//sorting allYears array
+	return allYears.sort(function(a,b){return a-b});
```
Comment (ankushdharkar) [inline NEWTASK1.js:84]: Why? Make this optional. You may be unnecessary adding time complexity to your solution

File: NEWTASK1.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+				}
+			];
+
+function yearWithMaxNumOfMovies(movieList){
+	
+	var maxYearList=[];
+
+	//Adding new key-value pair with all years of license
+	for(var i=0;i<movieList.length;i++){
+		movieList[i].year_range=range(movieList[i].license[0],movieList[i].license[1]);
```
Comment (ankushdharkar) [inline NEWTASK1.js:31]: Use destructured variable names for easy readability

**Comment by ankushdharkar** [comment]
In future, please make sure that your branch in your repo is named `develop`

**Comment by akash350poria** [comment]
Will get back to you after making all the necessary changes that you have suggested.

**Pull request closed without merging**